### DISCUSSION TITLE
slider for modal opacity

### DISCUSSION THREAD

**Discussion by greenthus** [vmware-clarity/ng-clarity#142] [Ideas]
in functional testing I sometimes need to see the data of the modal and in the background at the same time.

I need a slider that can arrange the opacity of the modal quickly.
("we make use of a developer login user")

![image](https://user-images.githubusercontent.com/104067056/171124422-2c70fe58-27a6-4502-b6b2-699018d5a62c.png)

**Comment by bbogdanov** [comment]
@greenthus What's stopping you to accomplish that?

**Reply by greenthus** [reply]
aaaaah I see your point now, I will discuss that with the developers (I'm a QA).

**Reply by greenthus** [reply]
Do you happen to know if fonts in 1 column are not the same size what could be the cause?
![image](https://user-images.githubusercontent.com/104067056/171407358-8c26057d-bc41-416c-93f5-3f2c63b3f268.png)

or should I make a bug report?
